.github/workflows/ci.yml

```diff
@@ -160,98 +139,32 @@ jobs:
           echo PYTEST_ADDOPTS="--basetemp C:/tmp/gha/pytest" >> ${GITHUB_ENV}
           echo SCIE_BASE=C:/tmp/gha/nce >> ${GITHUB_ENV}
       - name: Unit Tests
-        if: matrix.docker-image == ''
+        if: matrix.image == ''
         run: |
           "${UV}" run dev-cmd test -- -vvs
       - name: Unit Tests
-        if: matrix.docker-image != ''
+        if: matrix.image != ''
         run: |
-          cat << EOF > _test.sh
```

Comment:
All this yaml-thrash and the paired intro of `scripts/docker/uv.py` was just so I could test the install.sh changes locally via `uvrc docker -- --image alpine test -- -vvsk test_installer` (where `uvrc` is my alias for `uv run dev-cmd`). The upshot of streamlined yaml / an easy way to repro CI locally is, of course, generally good.